apps/geomat/src/app/app.module.ts

```diff
@@ -47,6 +48,7 @@ export class MyHammerConfig extends HammerGestureConfig {
     LandingComponent,
     LandingBannerContentComponent,
     InfoPageContentComponent,
+    CdkDragDropSortingExampleComponent,
```

Comment:
Why did you add this empty component. It is not needed. Please remove it.

Reply:
In the beginning, during trying to configure the solid-frontend project in my local machine, was facing backend API was not working and the menu tab of "selbtest" was not opening except loading the page. Then I created a new component "cdk-drag-drop-sorting-example" with a new menu "Drag and Drop". After discussing with "Lars Dietzel" backend access was working and then I fetch the code files than on the click menu tab "Slebtest" was working for me. So, I forget to remove the "CdkDragDropSortingExampleComponent" files and declaration. But I removed now all these unnecessary codes and declarations.